README.md

```diff
@@ -35,195 +26,101 @@ https://simplereport.gov/
-# If not, use POSTGRES_USER=postgres
-POSTGRES_USER=$USER LIB_DIR="$(pwd)/backend/db-setup" POSTGRES_DB=postgres ./backend/db-setup/create-db.sh
-```
-
-Then run this to start the app:
-
-```bash
-./start.sh
-```
-
-This will also start up both servers in "watch" mode. When using `start.sh`, any environment variables put
-in `.env` in the root directory will be available to the app. Press CTRL-C to exit and cleanup the servers cleanly.
-
-### Running locally with Okta
-
-You can run the app against the "Okta Preview" instance by running the backend with the `okta-local` Spring profile. Be sure to set the `OKTA_API_KEY` environment variable. You can generate an API token for yourself by logging into the Okta Preview [admin panel](https://hhs-prime-admin.oktapreview.com) and going into Security > API > Tokens.
-
-You also need to set the following in `frontend/.env.local`:
+1. Install [docker and docker-compose](https://docs.docker.com/get-docker/).
+1. Install [mkcert](https://github.com/FiloSottile/mkcert#installation)
+1. Install Node.js 14, either using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) or your preferred method
+1. Clone this repository: `git clone git@github.com:CDCgov/prime-simplereport.git`
+1. Run `yarn install` in the root of the repository to install pre-commit hooks using lefthook
+1. Run `cp .env.sample .env` in the root of the repository, then obtain the needed secrets from Azure or another developer. Be sure to set the `OKTA_API_KEY` environment variable. You can generate an API token for yourself by logging into the Okta Preview [admin panel](https://hhs-prime-admin.oktapreview.com) and going into Security > API > Tokens.
+1. Run `sudo vim /etc/hosts` (or use your editor of choice) and add a line to the bottom with the following contents and save: `127.0.0.1 localhost.simplereport.gov`
+1. Run `yarn start` in the root of the repository to start the app
+1. When the frontend and backend builds are complete, visit https://localhost.simplereport.gov/app
 
+#### Notes
+Make sure docker has access to enough resources on your local machine. You'll know it doesn't if you get an error from any of your containers similar to this:
 ```
-REACT_APP_BACKEND_URL=http://localhost:8080
-REACT_APP_BASE_URL=http://localhost:3000
-REACT_APP_OKTA_ENABLED=true
+"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
 ```
 
-### Updating user role
-
-By default the local test user is an organization admin role. If you need to change this value to test out other permissions.
-It can be set in `application-local.yaml`. If you have not created one run:
-
-bash
-
-```
-touch backend/src/main/resources/application-local.yaml
-```
+How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
```

Comment:
**Adding note based on pairing session**

On Mac, I needed to give Docker (and my terminal/VS Code for some reason?) access to my disk so it could perform the relevant filesystem operations. Nick mentioned this is might be because my code is housed within my Desktop folder, so leaving this for any future searchers.

Went to System Preferences > Security and Privacy > Privacy and scrolled down to Full Disk Access, granting the relevant apps access to my disk

<img width="660" alt="image" src="https://user-images.githubusercontent.com/29645040/154575811-addd130a-b456-49ba-be3d-446ff8427151.png">